src/map_ctrl.js

```diff
@@ -105,15 +108,15 @@ export class MapCtrl extends MetricsPanelCtrl {
     }
     if (this.points.length > 0) {
       if (this.panel.mode === this.modeTypes[0]) {
-        this.mapLayer = MapCtrl.L.hexbinLayer(this.panel.hexbin).hoverHandler(MapCtrl.L.HexbinHoverHandler.tooltip());
+        this.mapLayer = leaflet.hexbinLayer(this.panel.hexbin).hoverHandler(leaflet.HexbinHoverHandler.tooltip());
```

Comment:
@Drazair Have you checked what happens if that `leaflet` variable gets redefined globally later? I think it will lead to the same problem. I think it needs to be kept within a closure, or within the MapCtrl object

Reply:
It is being imported locally now, so there shouldn't be any problems. I tried it with importing as L (same as the global variable), but just changed to name to not confuse it with the global version of leaflet.